boards/arc/nsim_em/nsim_sem.yaml

```diff
@@ -1,7 +1,8 @@
-identifier: nsim_em
+identifier: nsim_sem
```

Comment:
em for emulator
sim for simuator
what's sem?

Reply:
my guess em is for ARC EM, sem for ARC SEM (S for Secure)  https://www.synopsys.com/dw/ipdir.php?ds=arc-sem